.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Description
```

Comment:
Is this `###` section style a convention from anywhere else? We tend to use "Section Name:" in most other places; the diff importer knows how to handle some of those sections, like "Test Plan:".

Our existing internal git tooling uses this as a commit template. Most of these aren't useful in a PR, but we can follow the pattern
```
<Replace this line with a title. Use 1 line only, 67 chars or less>

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:
```

Reply:
It's just a style choice. I like it since the format isn't too large. As an example, [Spark](https://github.com/apache/spark/blob/master/.github/PULL_REQUEST_TEMPLATE) uses three.

Reply:
Just confirmed that you don't need the colon for phabricator to correctly format the test plan